docs/source/install/monitoring_stack.rst

```diff
@@ -97,7 +100,7 @@ As an alternative, you can clone and use the Git repository directly.
 
    git clone https://github.com/scylladb/scylla-monitoring.git
    cd scylla-monitoring
-   git checkout branch-3.7
+   git checkout branch-|version|
```

Comment:
This substitution does not seems to work

Reply:
probably why it was left in the first place, It's now 3.8